src/highlight_processor.c

```diff
@@ -89,6 +89,16 @@ static size_t stripDuplicateSpaces(char *s, size_t n) {
   return oix;
 }
 
+/**
+ * Returns the length of the buffer without trailing spaces
+ */
+static size_t trimTrailingSpaces(const char *s, size_t input) {
+  for (; input && isspace(s[input - 1]); --input) {
+    // Nothing
```

Comment:
';' will do the trick ..

Reply:
I know, but it's easier to follow (IMO) if I just use an empty body explicitly.